src/components/provider/show/provider-show.js

```diff
@@ -280,6 +285,53 @@ class ProviderShow extends Component {
     );
   }
 
+  openEditProvider = () => {
+    const {
+      stripes,
+      onEdit,
+    } = this.props;
+
+    const editPermission = stripes.hasPerm(this.editPermissionName);
+
+    if (editPermission) {
```

Comment:
If this value returns the boolean so better to name it as `hasEditPermission` or `isEditPermission`